4001-5000/LIT4723TarikaWaldaAmid.xml

```diff
@@ -66,1219 +68,47 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+            <change when="2024-02-02" who="CH">Updated titles, abstract and bibliography</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
             <div type="bibliography">
                 <listRelation>
-               <relation name="saws:contains" active="LIT4723TarikaWaldaAmid" passive="LIT4089Introduction"/>
+                    <relation name="saws:contains" active="LIT4723TarikaWaldaAmid" passive="LIT4089Introduction"/>
                     <relation name="betmas:formerlyAlsoListedAs" active="LIT4723TarikaWaldaAmid" passive="LIT1076alMagm"/>
-               <relation name="dcterms:creator" active="LIT4723TarikaWaldaAmid" passive="PRS4711Giyorgis"/>
-               <relation name="saws:isVersionInAnotherLanguageOf" active="LIT4723TarikaWaldaAmid#t1" passive="LIT4723TarikaWaldaAmid#t2"/>
-            </listRelation>
+                    <relation name="dcterms:creator" active="LIT4723TarikaWaldaAmid" passive="PRS4711Giyorgis"/>
+                    <relation name="saws:isVersionInAnotherLanguageOf" active="LIT4723TarikaWaldaAmid#t1" passive="LIT4723TarikaWaldaAmid#t2"/>
+                </listRelation>
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Hoffmann2021Amid"/>
+                    </bibl>
+                </listBibl>
                 <listBibl type="secondary">
                     <bibl>
-                  <ptr target="bm:Kropp2016Zekra"/>
-                  <citedRange unit="page">1-12</citedRange>
-               </bibl>
+                        <ptr target="bm:Diez2024Amid"/><citedRange unit="page">140-141</citedRange>
+                    </bibl>
                     <bibl>
-                        <ptr target="bm:Witakowska2006Ethiopicuniversal"/>
-                        <citedRange unit="page">285-301</citedRange>
+                        <ptr target="bm:Witakowska2006Ethiopicuniversal"/><citedRange unit="page">285-301</citedRange>
                     </bibl>
-                    
                     <bibl>
-                  <ptr target="bm:Pietruschka2005WaldaAmid"/>
-               </bibl>
+                        <ptr target="bm:Pietruschka2005WaldaAmid"/>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:CahenCoquinEI1991Amid"/>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Kropp2016Zekra"/><citedRange unit="page">1-12</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Kropp1986ArabÄthÜbersetzungstechnik"/>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Kropp1989Haylu"/>
+                    </bibl>
                 </listBibl>
-            </div>
-            <div type="edition">
```

Comment:
Have you already replaced the relevant occurrences of LIT4723TarikaWaldaAmid with LIT6936TarWaAmidBeta?

(Just to make sure: You should not delete the structure before checking that there are no manuscript descriptions pointing to LIT4723TarikaWaldaAmid#WorldHistory LIT4723TarikaWaldaAmid#Preface etc etc
Since this edition follows the catalogue I assume these pointers have been used.)

Reply:
I was not aware of this problem. I will care about the links in the manuscript records as soon as possible. To my estimation, there should be a very limited number of occurrences, because most of the manuscripts containing the Tārika Walda ʾAmid are not yet properly described.

Reply:
I have noticed, that there is no link in LIT4089Introduction, that points to LIT4723TarikaWaldaAmid, so no link needs to be updated or replaced. Question is, whether it might be useful to point to the LIT4723TarikaWaldaAmid (or to LIT6936TarWaAmidBeta). Personally, I would recommend to obstain from that, because I think it is not certain, that this introduction is taken directly from Tarika Walda ʾAmid manuscript or rather from a collation or later paraphrase. As far as this connection is not investigated in detail, I prefer to keep it, as it is.

Reply:
I replaced the pointers in LIT4724WaldaAmidIntroduction and in LIT4839IntroductionWaldaAmid. However, I am not content with that structure, because I think, that the text in these manuscripts are rather incomplete witnesses of LIT6936TarWaAmidBeta. I may be able to give evidence to this assumption and rework the records FSUor40 and FSUor134 and likewise LIT4839IntroductionWaldaAmid and LIT4724WaldaAmidIntroduction in the future. For the moment, it might be OK, to keep it as it is.

Reply:
thank you, main thing is that the pointers in `ref `correspond to the `label`s